homeassistant/components/switch/onewire.py

```diff
@@ -125,17 +121,12 @@ def is_on(self):
         """Return true if switch is on."""
         return self._state
 
-    def update(self):
```

Comment:
Don't remove `update`. This is needed to read the updated state. Just don't return anything from `update`.

Reply:
Hi Martin,
I've accidentally removed update ! 
Thanks for your request - I've commited it again with the previous update function.
Sincerely,
Nik